### PR TITLE
Add tick API v2 resource for collecting page view stats

### PR DESCRIPTION
**Pull request by alex-mtl**
- Add new TickApiController to register basic pageviews
- Add openapi spec yml file for tick endpoint

Closes: https://github.com/vanilla/knowledge/issues/83

### CODE REVIEW

**Comment by alex-mtl** [comment]
@tburry Could you please confirm that this minimal implementation is good to go and we don't need to dive deeper and refactor `Gdn_Statistics` class?

File: applications/dashboard/openapi/tick.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        '201':
+          description: Success
+      tags:
+      - Statistics
+components:
+  schemas:
+    TickRequest:
+      description: Both body request parameters are optional
+      properties:
+        Path:
```
Comment (tburry) [inline applications/dashboard/openapi/tick.yml]: These should adhere to our naming conventions and be lower camel case.

File: applications/dashboard/openapi/tick.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+          description: Page path
+          minLength: 1
+          type: string
+          example: /discussions/123
+        ResolvedPath:
```
Comment (tburry) [inline applications/dashboard/openapi/tick.yml]: Can you leave this one out of the docs? It is kind of outdated and I'd rather new code doesn't depend on it.

File: applications/dashboard/openapi/tick.yml (hunk 1 above)
Comment (alex-mtl) [reply applications/dashboard/openapi/tick.yml]: I was trying to keep backward compatibility with old endpoint.

**Pull request merged**